src/Symfony/Component/DependencyInjection/Compiler/AttributeAutoconfigurationPass.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * @author Alexander M. Turek <me@derrabus.de>
+ */
+final class AttributeAutoconfigurationPass implements CompilerPassInterface
```

Comment:
In https://github.com/symfony/symfony/pull/39804/files#diff-109a013c942e91fd5e860bce2dab01b058e6de2d1fd712368b5e8a82e6618eabR30, I'm proposing a `'container.ignore_attributes'` tag to disable the introspection of attributes.

I think it would be useful here too.

Reply:
Good idea!